src/components/usersList/usersList.js

```diff
@@ -0,0 +1,26 @@
+import './usersList.scss';
+
+const CmpntName = ({ firstName = 'Philipp', lastName = 'Cherpakov', age = 26 }) => (
```

Comment:
`CmpntName` what about this name? How I can understand the main purpose of that function? Please, give a better name